osf/models/node.py

```diff
@@ -1860,28 +1860,24 @@ def fork_node(self, auth, title=None, parent=None):
         # Need to save here in order to access m2m fields
         forked.save()
 
+        forked.tags.add(*self.all_tags.values_list('pk', flat=True))
```

Comment:
Does this move have a functional effect?

Reply:
Nope, it is just being moved by solving merge conflict. Doesn't really change anything.